mmv1/templates/terraform/examples/bigquery_connection_kms.tf.erb

```diff
@@ -0,0 +1,24 @@
+resource "google_bigquery_connection" "<%= ctx[:primary_resource_id] %>" {
+   connection_id = "<%= ctx[:vars]['connection_id'] %>"
+   location      = "aws-us-east-1"
```

Comment:
It looks like this is causing the bigqueryconnection API to return the error:

```
Cloud KMS keys are not supported in region: aws-us-east-1
```

Generally we use us-central1 unless there's a specific reason not to (and it would match the bootstrapped key):

```suggestion
   location      = "us-central1"
```